x-pack/plugins/stack_alerts/public/rule_types/es_query/validation.ts

```diff
@@ -144,11 +144,12 @@ const validateCommonParams = (ruleParams: EsQueryRuleParams) => {
       })
     );
   }
-  if ((size && size < 0) || size > 10000) {
+  const maxSize = isServerless ? 100 : 10000;
```

Comment:
nit: can we set these in a const and reuse? I see there is already a const for the serverless max size

Reply:
Resolved in this commit [e2cb89f](https://github.com/elastic/kibana/pull/180520/commits/e2cb89f644da3687d86bf23526d15ab183ae7207)